cMake/FindMEDFile.cmake

```diff
@@ -41,6 +41,10 @@ IF(MEDFILE_ROOT_DIR)
 ENDIF(MEDFILE_ROOT_DIR)
 
 FIND_PATH(MEDFILE_INCLUDE_DIRS med.h PATH_SUFFIXES med)
+find_file(meddotH med.h PATHS ${MEDFILE_INCLUDE_DIRS} NO_DEFAULT_PATH)
+if(NOT meddotH)
+  message( FATAL_ERROR "med.h not found in an error message above.")
```

Comment:
Not sure what this message means?

Reply:
I followed the style already used in CMakeList.txt:
```
if(NOT Hdf5dotH)
  message( FATAL_ERROR "hdf5-serial not found in an error message above.")
endif()
```

Before this message appears, cmake find_file() have already output some message

Both perhaps both messages can be improved. I am open for suggestions about better phrasing, and it can be added into the same merge

Reply:
I am not sure what it must mean, that's why i didn't suggest anything... Indeed we can fix both then :) Maybe it means the file was not found AND there is more info in a message above? Anyone knows?

Reply:
The current output from cmake if med.h when the current branch is applied is:

```
-- Check for medfile (libmed and libmedc) ...
CMake Error at cMake/FindMEDFile.cmake:46 (message):
  med.h not found in an error message above.
Call Stack (most recent call first):
  CMakeLists.txt:706 (find_package)
```
FIND_PATH does not output an error on its own, and the error if not caught here, would appear later with this output

```
CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find MEDFile (missing: MEDFILE_INCLUDE_DIRS)
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cMake/FindMEDFile.cmake:58 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:706 (find_package)
```

New message should perhaps simply say:
med.h not found, please install development header-files for libmedc

Reply:
Sounds good to me!